catkin_ws/src/image_rotation/include/image_rotation/ImageAlgorithm.h

```diff
@@ -0,0 +1,17 @@
+#ifndef IMAGE_ALGORITHM_H
+#define IMAGE_ALGORITHM_H
+
+#include <opencv2/opencv.hpp>
+
+
+class ImageAlgorithm
+{
+  public:
+    ImageAlgorithm()
+    {
+    }
+    static void RotateImage(cv::Mat &image, double angle);
+
+};
+
+#endif
```

Comment:
EOL character missing. Comment for header guards missing

Reply:
fixed